spec/rack/test_spec.rb

```diff
@@ -446,7 +509,7 @@ def close
       expect(last_response['Content-Type']).to eq('text/html;charset=utf-8')
     end
 
-    it 'raises an error if no requests have been issued', focus: true do
+    it 'raises an error if no requests have been issued' do
```

Comment:
Same with spec/spec_helper.rb the line `config.filter_run_when_matching :focus`.

Reply:
I think removing it is quiet important, otherwise it might lead to unexpected and invalidly green tests by accident. So I would remove it asap, because I guess its a leftover and its not intended to be there.

Reply:
@scepticulous yes that's true. 

How about creating new 1 PR to modify the 2 lines at first before merging this PR?

```
config.filter_run_when_matching :focus
```

and

``` 
it 'raises an error if no requests have been issued', focus: true
```